.github/workflows/release.yml

```diff
@@ -24,29 +31,29 @@ jobs:
             const version = require(`${process.env.GITHUB_WORKSPACE}/package.json`).version;
             core.setOutput('repo', context.repo.repo);
             core.setOutput('version', version);
-            const time = new Date(Date.now()).toISOString().slice(0,10);
-            const titleLink = version.replace(/\./g,'') + '-' + time;
-            core.setOutput('titleLink', titleLink);
-            try {
-              var latestRelease = await github.repos.getLatestRelease({
-                owner: context.repo.owner,
-                repo: context.repo.repo
-              });
-              const latestTag = latestRelease.data.tag_name;
-              core.setOutput('commitLink', `compare/${latestTag}...v${version}`);
-            } catch (error) {
-              if (error.message === 'Not Found') {
-                core.setOutput('commitLink', `commits/v${version}`);
-              } else {
-                core.setFailed(error.message);
-              }
-            }
-            return true;
 
-      - name: Generate Package
+      - name: Push commit
         run: |
-          yarn install
-          yarn run build
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+          git add .
+          git commit -m "chore(release): :bookmark: release v${{steps.version.outputs.version}}"
+          git push https://x-access-token:${{secrets.GITHUB_TOKEN}}@github.com/${{github.GITHUB_REPOSITORY}}.git
```

Comment:
Better to have a git tag here. Previously `--skip.tag` was used in `standard-version`.